configuration-sample/ods-core.env.sample

```diff
@@ -239,6 +239,9 @@ DOCKER_REGISTRY=docker-registry.default.svc:5000
 # Example: hostA;hostB:8443;hostC:8443
 APP_DNS=192.168.56.101.nip.io
 
+# URI of the OpenShift API
+OPENSHIFT_API_URL=https://192.168.56.101:8443
```

Comment:
why needed, same info is below with _HOST

Reply:
real API and HOST URI might bedifferent... maybe we should change both to example close to reality like:
```
OPENSHIFT_API_URL=https://api.odsbox.lan:8443
```
and 
```
OPENSHIFT_CONSOLE_HOST=https://ocp.odsbox.lan:8443/console/project/
```
Should I update them?

Reply:
@stitakis - the second one is wrong above ... (the prov app - which needs the information - attaches `console/project`)

Reply:
yes, you are right, I edited the comment above when I noticed it.

Reply:
For consistency I would leave as it in (with an IP https://192.168.56.101:8443) because we have IP adresses for all URIs in the `ods-core.env.sample`, ok?

Reply:
ok